Doc/library/fileinput.rst

```diff
@@ -147,8 +147,7 @@ available for subclassing as well:
    :meth:`fileno`, :meth:`lineno`, :meth:`filelineno`, :meth:`isfirstline`,
    :meth:`isstdin`, :meth:`nextfile` and :meth:`close` correspond to the
    functions of the same name in the module. In addition it has a
-   :meth:`~io.TextIOBase.readline` method which returns the next input line,
-   and a :meth:`__getitem__` method which implements the sequence behavior.
+   :meth:`~io.TextIOBase.readline` method which returns the next input line.
```

Comment:
"```In addition it is :term:`iterable` and has a``` ..."

The only purpose of `__getitem__` was supporting iteration (before introducing the iterable protocol).